package.json

```diff
@@ -16,11 +17,31 @@
   },
   "author": "Jon Ursenbach <jon@readme.io>",
   "license": "ISC",
-  "dependencies": {},
+  "engines": {
+    "node": ">=10"
+  },
+  "dependencies": {
+    "@apidevtools/json-schema-ref-parser": "^9.0.1",
+    "@openapi-contrib/openapi-schema-to-json-schema": "^3.0.0",
+    "@readme/oas-examples": "^3.4.0",
```

Comment:
Lets shift over the correct ones to devDependencies that dont need to be real deps.